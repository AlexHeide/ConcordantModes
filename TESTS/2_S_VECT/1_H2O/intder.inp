# FILES #
intder

Initial Intder

# INTDER #2   3    4    5    6    7    8    9   10   11   12   13   14   15   16 #
    3    3    0    2    0 4234    0    0    0    1    3    0    0    0    0    0
STRE     2    1
STRE     3    2
BEND     3    2    1
      -0.0000000000        -1.4236653500        0.9926470200
       0.0000000000         0.0000000000       -0.1250915700
      -0.0000000000         1.4236653500        0.9926470200
           H           O           H
