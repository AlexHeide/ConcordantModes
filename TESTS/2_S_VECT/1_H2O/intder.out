
 INTDER2005

 A GENERAL PROGRAM DEVELOPED BY WESLEY D. ALLEN AND CO-WORKERS
 WHICH PERFORMS VARIOUS VIBRATIONAL ANALYSES AND HIGHER-ORDER
 NONLINEAR TRANSFORMATIONS AMONG FORCE FIELD REPRESENTATIONS

   PROGRAM OPTIONS

   NA      NSYM       NEQ      NINV     NTEST     NFREQ      NVEC     NDISP
        NS      NDER      NPRT      NDUM     NGEOM     IRINT     NSTOP     NMODE
    3    3    0    2    0 4234    0    0    0    1    3    0    0    0    0    0


 NUCLEAR CARTESIAN COORDINATES (BOHR)

         0.0000000000       -1.4236653500        0.9926470200
         0.0000000000        0.0000000000       -0.1250915700
         0.0000000000        1.4236653500        0.9926470200


 DEFINITION OF INTERNAL COORDINATES

 SIMPLE INTERNAL COORDINATES

L( 1)=   STRE    2    1    0    0     
L( 2)=   STRE    3    2    0    0     
L( 3)=   BEND    3    2    1    0     



 VALUES OF SIMPLE INTERNAL COORDINATES (ANG. OR DEG.)

   1    0.9578199693   2    0.9578199693   3  103.7281405857

 B MATRIX FOR (SYMMETRY) INTERNAL COORDINATES

 I=            1
    0.000000   -0.786548    0.617529    0.000000    0.786548   -0.617529
    0.000000    0.000000    0.000000
 I=            2
    0.000000    0.000000    0.000000    0.000000   -0.786548   -0.617529
    0.000000    0.786548    0.617529
 I=            3
    0.000000   -0.644724   -0.821186    0.000000    0.000000    1.642371
    0.000000    0.644724   -0.821186


 B*BT MATRIX FOR (SYMMETRY) INTERNAL COORDINATES

 I=           1
    2.000000   -0.237315   -1.014212
 I=           2
   -0.237315    2.000000   -1.014212
 I=           3
   -1.014212   -1.014212    4.877412

 DETERMINANT OF B*BT MATRIX=  14.632    


 A MATRIX FOR (SYMMETRY) INTERNAL COORDINATES

 I=           1
    0.000000    0.000000    0.000000
 I=           2
   -0.569053   -0.217495   -0.295741
 I=           3
    0.240928   -0.035085   -0.125562
 I=           4
    0.000000    0.000000    0.000000
 I=           5
    0.351559   -0.351559   -0.000000
 I=           6
   -0.205843   -0.205843    0.251124
 I=           7
    0.000000    0.000000    0.000000
 I=           8
    0.217495    0.569053    0.295741
 I=           9
   -0.035085    0.240928   -0.125562


 FINAL INTERNAL (SYMMETRY) COORDINATE FORCE CONSTANTS

 QUADRATIC FORCE CONSTANTS

    8.888080   -0.129034    0.261824
   -0.129034    8.888080    0.261824
    0.261824    0.261824    0.708028


 NORMAL MODE ANALYSIS IN INTERNAL COORDINATES


 NUCLEAR CARTESIAN COORDINATES (ANG.) AND MASSES

  ATOM    MASS             X             Y             Z

    1    1.007825     0.0000000000 -0.7533713134  0.5252862193
    2   15.994915     0.0000000000  0.0000000000 -0.0661956129
    3    1.007825     0.0000000000  0.7533713134  0.5252862193


 QUADRATIC FORCE CONSTANTS IN MDYN/A, MDYN/RAD, OR MDYN*A/RAD**2

    1  1  8.888080    1  2 -0.129034    1  3  0.261824
    2  1 -0.129034    2  2  8.888080    2  3  0.261824
    3  1  0.261824    3  2  0.261824    3  3  0.708028



 LOWEST EIGENVALUES OF THE G MATRIX

    1.033724    1.069593    2.337938



 VIBRATIONAL FREQUENCIES (CM-1) AND EIGENVECTORS


 ----------------------------------------
                1         2         3
 COORDINATE    1655.301    3925.457    4045.924
 ----------------------------------------
      1     -0.0465350605  0.7195789325 -0.7312976637
      2     -0.0465350605  0.7195789325  0.7312976637
      3      1.5269674208  0.0106299776 -0.0000000000
 ----------------------------------------





 TOTAL ENERGY DISTRIBUTIONS (%)


 ----------------------------------------
                1         2         3
 COORDINATE  1655.301  3925.457  4045.924
 ----------------------------------------
      1       0.02250  49.97750  50.00000
      2       0.02250  49.97750  50.00000
      3      99.95500   0.04500  -0.00000
 ----------------------------------------





 VIBRATIONAL ASSIGNMENTS


   MODE    CM-1                 DOMINANT COMPONENTS OF TED

     1    1655.3      3 (100.0)
     2    3925.5      1 ( 50.0)    2 ( 50.0)
     3    4045.9      2 ( 50.0)   -1 ( 50.0)


 NORMAL MODE ANALYSIS IN CARTESIAN COORDINATES


 NUCLEAR CARTESIAN COORDINATES (ANG.) AND MASSES

  ATOM    MASS             X             Y             Z

    1    1.007825     0.0000000000 -0.7533713134  0.5252862193
    2   15.994915     0.0000000000  0.0000000000 -0.0661956129
    3    1.007825     0.0000000000  0.7533713134  0.5252862193


 QUADRATIC FORCE CONSTANTS IN HARTREE/BOHR**2

    1  1  0.000000    1  2  0.000000    1  3  0.000000    1  4  0.000000    1  5  0.000000    1  6  0.000000
    1  7  0.000000    1  8  0.000000    1  9  0.000000
    2  1  0.000000    2  2  0.389142    2  3 -0.249044    2  4  0.000000    2  5 -0.358310    2  6  0.216775
    2  7  0.000000    2  8 -0.030832    2  9  0.032269
    3  1  0.000000    3  2 -0.249044    3  3  0.231314    3  4  0.000000    3  5  0.281314    3  6 -0.241764
    3  7  0.000000    3  8 -0.032269    3  9  0.010451
    4  1  0.000000    4  2  0.000000    4  3  0.000000    4  4  0.000000    4  5  0.000000    4  6  0.000000
    4  7  0.000000    4  8  0.000000    4  9  0.000000
    5  1  0.000000    5  2 -0.358310    5  3  0.281314    5  4  0.000000    5  5  0.716620    5  6  0.000000
    5  7  0.000000    5  8 -0.358310    5  9 -0.281314
    6  1  0.000000    6  2  0.216775    6  3 -0.241764    6  4  0.000000    6  5  0.000000    6  6  0.483529
    6  7  0.000000    6  8 -0.216775    6  9 -0.241764
    7  1  0.000000    7  2  0.000000    7  3  0.000000    7  4  0.000000    7  5  0.000000    7  6  0.000000
    7  7  0.000000    7  8  0.000000    7  9  0.000000
    8  1  0.000000    8  2 -0.030832    8  3 -0.032269    8  4  0.000000    8  5 -0.358310    8  6 -0.216775
    8  7  0.000000    8  8  0.389142    8  9  0.249044
    9  1  0.000000    9  2  0.032269    9  3  0.010451    9  4  0.000000    9  5 -0.281314    9  6 -0.241764
    9  7  0.000000    9  8  0.249044    9  9  0.231314



 ZERO FREQUENCIES (CM-1)

     -0.0000      0.0000      0.0000      0.0000      0.0121      0.0338



 VIBRATIONAL FREQUENCIES (CM-1) AND EIGENVECTORS


 ----------------------------------------
                1         2         3
 COORDINATE    1655.301    3925.457    4045.924
 ----------------------------------------
      1      0.0000000000  0.0000000000  0.0000000000
      2     -0.4149846884  0.5691270386 -0.5336000753
      3     -0.5363354689 -0.3910743865  0.4189365144
      4      0.0000000000 -0.0000000000  0.0000000000
      5     -0.0000000000  0.0000000000  0.0672433078
      6      0.0675880171  0.0492824805 -0.0000000000
      7     -0.0000000000  0.0000000000 -0.0000000000
      8      0.4149846884 -0.5691270386 -0.5336000753
      9     -0.5363354689 -0.3910743865 -0.4189365144
 ----------------------------------------





   MODE    CM-1

     1    1655.3
     2    3925.5
     3    4045.9
